electron/generators/diograph-generator.js

```diff
@@ -58,12 +64,12 @@ async function generateDiograph(folderPath) {
   const rootDiory = generateDioryFromFolder(folderPath, dioryLinks)
 
   return {
-    key: basename(folderPath),
+    key: 'string', // basename(folderPath),
     id: rootDiory.id,
     diograph: {
-      ...reduceDiorysToDiograph([rootDiory]),
-      ...reduceDiorysToDiograph(Object.values(fileDioryLinks)),
-      ...reduceSubfolderDiographsToDiograph(subfolderDiographs),
+      // ...reduceDiorysToDiograph([rootDiory]),
+      // ...reduceDiorysToDiograph(Object.values(fileDioryLinks)),
+      // ...reduceSubfolderDiographsToDiograph(subfolderDiographs),
```

Comment:
Näitä funkkareita ei ole vielä olemassa enkä ihan pysynyt kartalla, miten tämä diograph luotaisiin tai miten olit sen ajatellut...